Lexplorer/Components/HomepageOverview.razor

```diff
@@ -1,10 +1,10 @@
 ﻿@using Lexplorer.Helpers
 @using Lexplorer.Models
 @using System.Diagnostics
-@inject Lexplorer.Services.LoopringGraphQLService LoopringGraphQLService;
+@*@inject Lexplorer.Services.LoopringGraphQLService LoopringGraphQLService;
 @inject Lexplorer.Services.UniswapGraphQLService UniswapGraphQLService;
 @inject NavigationManager NavigationManager;
-@inject IAppCache AppCache;
+@inject IAppCache AppCache;*@
 
```

Comment:
What's it with these extra `*@`? Was this intentional?

Reply:
Never mind - this was just commenting it out.

Reply:
That's just a typo :) Will remove it